src/design/plone/contenttypes/restapi/services/types/get.py

```diff
@@ -86,22 +86,46 @@ class FieldsetsMismatchError(Exception):
         "dates",
         "categorization",
     ],
-    "Venue": ["default", "informazioni", "contatti", "categorization"],
+    "Venue": ["default", "dove", "contatti", "informazioni", "categorization"],
 }
 
 
 @implementer(IPublishTraverse)
 class TypesGet(BaseGet):
     def customize_persona_schema(self, result):
         msgid = _(u"Nome e Cognome", default="Nome e cognome")
-        result["properties"]["title"]["title"] = translate(msgid, context=self.request)
+        result["properties"]["title"]["title"] = translate(
+            msgid, context=self.request
+        )
+        return result
+
+    def customize_luogo_schema(self, result):
+
+        if "fieldsets" in result:
```

Comment:
questo non riesci a farlo ordinando i campi/behaviors? Vorrei tenere al minimo le personalizzazioni qui

Reply:
no, mi si richiede di mischiare  campi definiti in schema (S) e campi di behavior (B)
titolo, (B)
nome alternativo (S)
descrizione (B)
immagine in testata (B)
descrizione completa (S)

Reply:
se sai un modo alternativo ben accetto, ma non ho trovato altre soluzioni

Reply:
qualcosa tipo questa nello schema della behavior luogo non funziona?

`
    form.order_after(nome_alternativo="IBasic.title")
    form.order_after(descrizione_completa ="ILeadImageBehavior.image_caption")
`

Reply:
si, ma da una certa in poi devo iniziare ad ordinarli in senso contrario per avere leadimage dove si vuole. Si fa eh, ma mi faceva leggermente schifo. Poi se a qualcuno viene in mente di cambiarli, invece che cambiare una lista di stringhe bisogna rifarsi l'ordine sullo schema

Reply:
E' che questa è la plone-way e si riflette anche nel backend. Se lo modifichi solo nella rotta, si vede solo li.

Comunque puoi anche fare così, che forse è anche più intuibile:

    form.order_after(nome_alternativo="IBasic.title")
    form.order_after(descrizione_completa="ILeadImageBehavior.image_caption")
    form.order_after(elementi_di_interesse="descrizione_completa")
    form.order_after(modalita_accesso="elemenit_di_interesse")
    form.order_after(orario_pubblico="modalita_accesso")

Reply:
non riesco a sistemarci in mezzo i campi. di leadimage cosi', come ho fatto nell'ultimo commit va bene invece

Reply:
So che sono pesante, ma ho provato in locale, su un sito nuovo e l'ordinamento ce l'ho corretto ( title, nome_alternativo, description, image, image_caption, descrizione_completa, elementi_di_interesse, modalita_accesso, orario_pubblico) anche solo con queste 3 righe:

    form.order_after(circoscrizione="IGeolocatable.coordinates")
    form.order_after(quartiere="IGeolocatable.coordinates")
    form.order_after(nome_alternativo="IBasic.title")

Le successive non servono perché la behavior "luogo" è stata messa dopo a quella della leadimage nella lista delle behavior.